editor/src/components/editor/actions/action-creators.ts

```diff
@@ -775,7 +772,7 @@ export function wrapInView(
 
 export function wrapInElement(
   targets: Array<ElementPath>,
-  whatToWrapWith: { element: JSXElement; importsToAdd: Imports },
+  whatToWrapWith: { element: JSXElement; importsToAdd: Imports } | 'conditional',
```

Comment:
This is a genuine question and not criticism: have you thought about just using `JSXElementLike | JSXConditionalExpression` instead of treating conditionals as a special case?
Maybe even fragments could mostly work out of the box that way

Reply:
I think it's a good point, the original reasoning was that conditionals (or fragments, since they're also a good mention) don't need any "element" per se as it'll never change, but indeed with the current situation where reparenting takes care of everything I think it makes sense to do what you're suggesting 👍